#Define the goal

uint8 POSE_TYPE = 1
uint8 PATH_TYPE = 2
uint8 type
geometry_msgs/Pose2D   target_pose
geometry_msgs/Pose2D[] target_path

---
#Define the result
bool success

---

#Define a feedback message
float32 speed
float32 steer_angle
float32 percentage
